server.js

```diff
@@ -5,49 +5,32 @@ const app = express();
 app.use(express.json());
 app.use(cors());
 app.set('port', process.env.PORT || 3001);
-const { Client } = require('pg');
-const pool = require("./db")
+const { Pool } = require('pg');
 
-var connectionString = "postgres://postgres:postgres@localhost:5432/database";
-const client = new Client({
-    connectionString: connectionString
-});
+const pool = new Pool({
+  connectionString: 'postgres://yjuallbecyquuz:ae9683fc25ec009cd257b0d1bcd71ba9efb41622fa4bd4e229336e67bd88105d@ec2-52-45-183-77.compute-1.amazonaws.com:5432/d89ngl6v155i5a',
```

Comment:
We are still missing some sort of certification in our connection from express to the database. I know this might be fixed when we deploy to heroku, but it would be nice to understand why this doesn't work on our local machines